zulipterminal/model.py

```diff
@@ -126,7 +126,10 @@ def __init__(self, controller: Any) -> None:
 
         self.unread_counts = classify_unread_counts(self)
 
-        self.emoji_data = emoji_data.EMOJI_DATA
+        custom_emojis = self.fetch_custom_emojis()
+        all_emojis = (list(emoji_data.EMOJI_DATA.items())
+                      + list(custom_emojis.items()))
+        self.emoji_data = OrderedDict(sorted(all_emojis, key=lambda e: e[0]))
```

Comment:
Is this really `active_emoji_data`, given the qualifier on the custom emojis?

Reply:
The active qualifier is only on custom emoji, but this contains `unicode_emoji` too?

Reply:
Do we consider `unicode_emojis` active by default?

Reply:
I'm not sure we can deactivate unicode emoji, though I see what you mean. At least partially I was trying to make the variable name obvious in the sense that if a custom emoji was deactivated subsequent to this point, then we might think to check - in the same way that `stream_dict` doesn't really describe `active_stream_subscriptions`.